cmd/broker/main.go

```diff
@@ -23,12 +24,20 @@ func main() {
 
 	configFilepath := os.Args[1]
 	servicePath := os.Args[2]
+
 	state, err := config.NewState(configFilepath, servicePath)
 	if err != nil {
 		logger.Fatal("main", err)
 	}
 
-	broker := broker.NewBlockheadBroker(state)
+	cli, err := client.NewEnvClient()
```

Comment:
Does this work with our docker file?

Reply:
probably more useful as a step to compile than to perform the operation. Eventually it will need to be in a container for kube, so that's a differnt restriction. Not sure how to proceed right now. Can take it out or README, but I do want to keep the dockerfile for later.

Reply:
@swetharepakula good point regarding our docker image. It won't work. 

A temp solution would be to have the deamon run inside the container just like how we do it in the CI. thoughts?

Reply:
yes, i think that makes sense. we need docker in docker. but that will require some changes with the dockerfile.